lib/teams-api.js

```diff
@@ -23,3 +35,39 @@ export async function createTeam (data) {
     }
   })
 }
+
+/**
+ * getTeam
+ * Get team details from the API
+ *
+ * @param id - id of team
+ * @returns {Object} - team details + moderators + members
+ */
+export async function getTeam (id) {
+  let res = await fetch(join(URL, id))
+  if (res.status === 200) {
+    return res.json()
+  } else {
+    const err = new Error('could not retrieve team')
+    err.status = res.status
+    throw err
+  }
+}
+
+/**
+ * updateTeam
+ * Update a team
+ *
+ * @param id - Team id
+ * @param values - Basic details { bio, name, hashtag }
+ * @returns {Response}
+ */
+export async function updateTeam (id, values) {
+  return fetch(join(URL, id), {
```

Comment:
`join` fails when `id` is a number because it is expecting a string.

Reply:
woops sorry about that. Fix incoming

Reply:
@kamicut we could either do `String(id)` in the couple of places where it is necessary or revise the `URL` var and do something like: ```join(URL, `teams/${id}`)```

Reply:
oh nvm you fixed it.